wireapp/wire-android
# List loading

Removes the `IncomingMessages` dependencies and pauses updating the conversation list if there are lots of events being processed in the background.

Depends on:
https://github.com/wireapp/wire-android-sync-engine/pull/173

To test: 
 - turn off the wifi/internet connection on the device
 - receive a lot of notifications in lots of different conversations (> 50)
 - turn the wifi back on
 - you should be able to scroll and go into conversations, but the conversation list will only update once (this may take up to 15 seconds!)
 - if it takes longer than 2 seconds - you should see the loading bar 
#### APK
[Download build #9101](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9101/artifact/build/artifact/wire-dev-PR964-9101.apk)
[Download build #9102](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9102/artifact/build/artifact/wire-dev-PR964-9102.apk)
[Download build #9106](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9106/artifact/build/artifact/wire-dev-PR964-9106.apk)
[Download build #9113](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9113/artifact/build/artifact/wire-dev-PR964-9113.apk)
